.github/workflows/import-base-images.yml

```diff
@@ -60,6 +60,13 @@ jobs:
       - name: Sync images from media library (git submodule) to this repo
         run: ./scripts/sync-base-images.sh
 
+      - name: Resize synched imaged
+        uses: ./.github/actions/resize-image
+        with:
+          images_path: public/images
```

Comment:
Maybe we should add another improvement on the README. We should process only new or modified images instead of all images.